uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/ui/carddetailscheckout/CardDetailsCheckoutScreen.kt

```diff
@@ -277,31 +276,34 @@ private fun CvvField(
     viewModel: CardDetailsCheckoutViewModel,
 ) {
     val scrollOffset = with(LocalDensity.current) {
-        if (state.isEmailInputFieldRequired && state.isPostalCodeFieldRequired) {
+        if (state.isBillingCountryFieldRequired && state.isPostalCodeFieldRequired) {
             FIFTH_FIELD_OFF_SET_DP.dp.toPx()
-        } else if (state.isPostalCodeFieldRequired || state.isEmailInputFieldRequired) {
+        } else if (state.isBillingCountryFieldRequired || state.isPostalCodeFieldRequired) {
             FORTH_FIELD_OFF_SET_DP.dp.toPx()
         } else {
             THIRD_FIELD_OFF_SET_DP.dp.toPx()
         }
     }
-
+    var focusedTextKey by remember { mutableStateOf(false) }
```

Comment:
Can be renamed to something that actually means what we want, like `hasBeenFocused`